src/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.cpp

```diff
@@ -0,0 +1,79 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.hpp"
+
+#include <cmath>
+#include <pup.h>  // IWYU pragma: keep
+
+#include "DataStructures/DataVector.hpp"     // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+// IWYU pragma: no_forward_declare Tensor
+
+namespace Elasticity {
+namespace Solutions {
+
+BentBeam::BentBeam(double length, double height, double bending_moment,
+                   constitutive_relation_type constitutive_relation) noexcept
+    : length_(length),
+      height_(height),
+      bending_moment_(bending_moment),
+      constitutive_relation_(std::move(constitutive_relation)) {}
+
+tuples::TaggedTuple<Tags::Displacement<2>> BentBeam::variables(
+    const tnsr::I<DataVector, 2>& x,
+    tmpl::list<Tags::Displacement<2>> /*meta*/) const noexcept {
+  const double youngs_modulus = constitutive_relation_.youngs_modulus();
+  const double poisson_ratio = constitutive_relation_.poisson_ratio();
+  const double prefactor =
+      3. * bending_moment_ / (2. * youngs_modulus * pow<3>(height_));
+  return {tnsr::I<DataVector, 2>{
+      {{-prefactor * get<0>(x) * get<1>(x),
+        prefactor / 2. *
+            (pow<2>(get<0>(x)) + poisson_ratio * pow<2>(get<1>(x)) -
```

Comment:
Why not `square()` instead of `pow<2>`? Here and throughout.

Reply:
sure, changed it